macros.cfg

```diff
@@ -212,11 +212,13 @@ gcode:
   M83
   # Lift to start print Z height
   G0 Z{z} F{z_speed}
-  # move to blob position along the edge of the bed
+  # move close to blob position along the edge of the bed
   G1 X{x_start} F{speed}
-  G1 Y{y_start} F{speed}
-  # Lower to blob height
+  G1 Y{y_start + 15} F{speed}
```

Comment:
This would fail in case `nozzle_prime_start_y` is `max` or anything larger than `axis_maximum.y - 15`.

Change to `G1 Y{y_start + (15 * y_factor)} F{speed}`

Reply:
Ah, didn't even think of that! Will fix, just a sec.